src/components/Swap/Input.tsx

```diff
@@ -33,10 +33,21 @@ export const Balance = styled(ThemedText.Body2)`
   transition: color 0.25s ease-in-out;
 `
 
-const InputColumn = styled(Column)<{ approved?: boolean }>`
-  margin: 0.75em;
+export const InputColumn = styled(Column)<{ approved?: boolean; hasColor?: boolean | null }>`
+  background-color: ${({ theme }) => theme.module};
+  border-radius: ${({ theme }) => theme.borderRadius - 0.25}em;
+  padding: 20px 0;
+  margin-bottom: 4px;
   position: relative;
 
+  // Set transitions to reduce color flashes when switching color/token.
```

Comment:
i'm assuming this comment will get removed right? idk how the styled components pipeline works